app/views/play/level/PlayLevelView.coffee

```diff
@@ -47,6 +47,9 @@ CourseVictoryModal = require './modal/CourseVictoryModal'
 PicoCTFVictoryModal = require './modal/PicoCTFVictoryModal'
```

Comment:
We should do changes related to `playLevelView` in the new file which is in ozaria folder: https://github.com/codecombat/codecombat/blob/master/ozaria/site/views/play/level/PlayLevelView.js

Reply:
I will do this! There seems to be a complication here though, where game dev levels are currently having an issue for the new `playLevelView`.